std/json.d

```diff
@@ -1160,7 +1161,7 @@ string toJSON(const ref JSONValue root, in bool pretty = false, in JSONOptions o
                 case '\t':      json.put("\\t");        break;
                 default:
                 {
-                    import std.uni : isControl;
+                    import std.ascii : isControl;
                     import std.utf : encode;
 
```

Comment:
I'd like to assert here that `escapeNonAsciiChars` is not set together with `Char = char`. If that could happen, we'd output a \u sequence for each `char` in a multibyte sequence (which would be completely wrong).